### PR TITLE
allow free tx as default on invocations

### PR DESCRIPTION
**Pull request by igormcoelho**
I think that it's not necessary that neo-cli enforces a minimum fee of 0.001 on all transactions, only on those that exceed 1024 bytes.

### CODE REVIEW

**Comment by shargon** [comment]
give me a couple of hours for check the gas consumption

**Comment by shargon** [comment]
https://github.com/neo-project/neo-plugins/blob/master/SimplePolicy/SimplePolicyPlugin.cs#L102 Good :)

File: neo-cli/Shell/MainService.cs [hunk 1]
```diff
@@ -283,10 +283,11 @@ private bool OnInvokeCommand(string[] args)
 
         public InvocationTransaction DecorateInvocationTransaction(InvocationTransaction tx)
         {
-            Fixed8 fee = Fixed8.FromDecimal(0.001m);
+            Fixed8 fee = Fixed8.FromDecimal(0);
```
Comment (shargon) [inline neo-cli/Shell/MainService.cs]:
```suggestion
            Fixed8 fee = Fixed8.Zero;
```

Comment (igormcoelho) [reply neo-cli/Shell/MainService.cs]: Committed change.

**Comment by vncoelho** [comment]
How is this going? Can you test this, @shargon ? I can set the environment for testing here as well.

**Comment by igormcoelho** [comment]
Thanks for testing @shargon, I'll try to improve testing capabilities of neo-cli in a next commit.

**Comment by igormcoelho** [comment]
@erikzhang a more complete testing structure is proposed here: https://github.com/neo-project/neo-cli/pull/382

**Pull request merged**